From 0e02cd1ceca5a73613a19a004d2c89a70a584a3c Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 12:58:34 -0400
Subject: [ethernet] qlge: Update version to 1.00.00.33

Message-id: <1408625927-3701-4-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88526
O-Subject: [PATCH RHEL 7.1 03/16] qlge: Update version to 1.00.00.33
Bugzilla: 1089357
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jitendra Kalsaria <jitendra.kalsaria@qlogic.com>

Bug 1089357 (https://bugzilla.redhat.com/show_bug.cgi?id=1089357)

net commit id 146669a5ab99285e95e2a7d04d8754534b585dd4

Signed-off-by: Jitendra Kalsaria <jitendra.kalsaria@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlge/qlge.h b/drivers/net/ethernet/qlogic/qlge/qlge.h
index 7e8d682..62c630b 100644
--- a/drivers/net/ethernet/qlogic/qlge/qlge.h
+++ b/drivers/net/ethernet/qlogic/qlge/qlge.h
@@ -18,7 +18,7 @@
  */
 #define DRV_NAME   "qlge"
 #define DRV_STRING  "QLogic 10 Gigabit PCI-E Ethernet Driver "
-#define DRV_VERSION "v1.00.00.32"
+#define DRV_VERSION "1.00.00.33"
 
 #define WQ_ADDR_ALIGN 0x3 /* 4 byte alignment */
 
-- 
1.7.1